drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -54,10 +65,29 @@ local device_type_profile_map = {
 }
 local detect_matter_thing
 
+local function get_field_for_endpoint(device, field, endpoint)
+  return device:get_field(string.format("%s_%d", field, endpoint))
+end
+
+local function set_field_for_endpoint(device, field, endpoint, value, additional_params)
+  device:set_field(string.format("%s_%d", field, endpoint), value, additional_params)
+end
+
 local function convert_huesat_st_to_matter(val)
   return math.floor((val * 0xFE) / 100.0 + 0.5)
 end
 
+local function mired_to_kelvin(value)
+  local CONVERSION_CONSTANT = 1000000
```

Comment:
Sorry I was a bit late on the review and I realize this has already been merged, but I do have a few comments.

`CONVERSION_CONSTANT` is already defined globally so we don't need to redefine it here. It'd be good to also give it a more specific name to indicate what it's for.